spynnaker/pyNN/models/utility_models/delays/delay_extension_machine_vertex.py

```diff
@@ -230,7 +231,7 @@ def generate_data_specification(self, spec, placement):
         # Reserve memory:
         spec.comment("\nReserving memory space for data regions:\n\n")
 
-        # ###################################################################
+        # ###################################################################self
```

Comment:
Looks like a "self" has been mistakenly added to the end of this line

Reply:
Oops - removed!